### PR TITLE
added instructions for wsl2 in beginning

### PR DESCRIPTION
**Pull request by markkyaw**
- Added toc for faster navigation
- Added instructions to enable WSL 2 in the beginning for windows users
- Added docker desktop screenshot of enabling WSL 2 integration
- Minor formatting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,4 +278,5 @@ wsl --setdefault <process name>
 
 After it finishes changing, open your Docker Desktop and navigate `Settings > Resources > WSL Integration`, make sure you check "Enable integration with my default WSL distro", and enable integrations with any of the additional distros you have. You'll have to restart your docker by doing this operation.
 
+![Docker Desktop WSL image](assets/docker-home.png)
```
Comment (camerondurham) [inline README.md]: I might be missing something but what was this image showing? The "Enable WSL2 integration", right? If so, do you mind adding a comment or referencing it in the preceeding paragraph so people know what it's for?

**Review by camerondurham — COMMENTED** [review]
Great additions and formatting changes to the README! Really appreciate the clarity added. Just had a minor question.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,25 @@
 
 <br>
 
+- [CSCI 350 Operating Systems Image](#csci-350-operating-systems-image)
```
Comment (camerondurham) [inline README.md:17]: Do you think this toc is more useful/explicit than the one github includes? People might not know about it but gh recently did add a toc menu bar to md files: 
image

Comment (markkyaw) [reply README.md:17]: Well I didn't know gh has toc baked in either LOL so maybe it's fine until more people start to know it exists

Comment (camerondurham) [reply README.md:17]: Yeah, this definitely makes it more apparent I'm cool with that.

File: README.md [hunk 3]
```diff
@@ -255,6 +277,9 @@ You can change the default distro by running
...
 ```
 
-After it finishes changing, open your Docker Desktop and navigate `Settings > Resources > WSL Integration`, make sure you check "Enable integration with my default WSL distro", and enable integrations with any of the additional distros you have. You'll have to restart your docker by doing this operation.
+After it finishes changing, open your Docker Desktop and enable `Use the WSL 2 based engine`. Then navigate to `Settings > Resources > WSL Integration`, make sure you check `Enable integration with my default WSL distro`, and enable integrations with any of the additional distros you have. You'll have to restart your docker by doing this operation.
+
+#### Docker Desktop Setting Screenshots
 
-![Docker Desktop WSL image](assets/docker-wsl.png)
+![Docker Desktop General Settings](assets/docker-home.png)
+![Docker Desktop Resources Settings](assets/docker-wsl.png)
```
Comment (camerondurham) [inline README.md:290]: Did you mean to remove this line and/or add the `docker-wsl.png` to the commit? I think it's missing from the PR somehow? 

image

**Review by camerondurham — COMMENTED** [review]
Instructions look good now! Sorry, one last small comment about the missing image/extra image.

Otherwise LGTM!

File: README.md (hunk 3 above)
Comment (camerondurham) [reply README.md:290]: Nevermind, resolved offline because I cannot see what's in front of me. 🙈

**Pull request merged**